src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImplTest.java

```diff
@@ -630,4 +579,33 @@ private RoleAdditionResponse createAddRoleResponse(HttpStatus status, String mes
         deleteRoleResponses.add(deleteRoleResponse);
         return deleteRoleResponses;
     }
+
+    @Test
+    public void modify_user_roles_feign_error_with_no_status() throws Exception {
+
+        when(feignExceptionMock.status()).thenReturn(-1);
+        callModifyRolesForUser(HttpStatus.INTERNAL_SERVER_ERROR);
+        verify(feignExceptionMock, times(1)).status();
+
+    }
+
+    @Test
+    public void modify_user_roles_feign_error_with_no_400_status() throws Exception {
+
+        when(feignExceptionMock.status()).thenReturn(400);
+        callModifyRolesForUser(HttpStatus.BAD_REQUEST);
+        verify(feignExceptionMock, times(2)).status();
+
+    }
+
+    public void callModifyRolesForUser(HttpStatus status) {
+        when(userProfileFeignClient.modifyUserRoles(any(), any(), any())).thenThrow(feignExceptionMock);
```

Comment:
add annotation @Test here

Reply:
This is not test. A common method used in couple of test cases.